README.md

```diff
@@ -1,8 +1,69 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Sai Prateek Reddy Annaiahgari . Jarvis Consulting
 
+I am a gradute of Electronics and Communication Engineering program with a specialization in Embedded Systems from Jawaharlal Nehru Technological Univarsity, Hyderabad, India. With a strong foundation in basic programming principles and analytical skills, I decided to explore the concepts of software engineering and expand my technical knowledge to pursue a career in Software Development. I have utilized various free online resources to learn different programming languages like python, C#, Java and Scala, and various tools by Apache Software Foundation. The improved knowledge helped me secure a job at Jarvis as a Data Engineer. At Jarvis, I learned to work as a team in a dynamic environment and develop efficient sofware solutions to complex real-world business problems. As a result of my abilty to quickly learn new technologies in a dynamic field like Data engineering, I will be a great fit to an equally dynamic team and organisation as a Data Engineer.
```

Comment:
please run a Grammarly check for the entire document

Reply:
remove this `I am a gradute of Electronics and Communication Engineering program with a specialization in Embedded Systems from Jawaharlal Nehru Technological Univarsity, Hyderabad, India.` for two reasons. FIrst, it's not CS or computer engineering. Second, it's not a Canadian university.

Reply:
I'm a recent new graduate with strong foundation in ....

Reply:
~I have utilized various free online resources to learn different programming~

I have experience in different programming languages ...

Reply:
~At Jarvis, I learned to work as a team I~

At Jarvis, I have worked as a team ...

Reply:
organisation as a ~Data Engineer~. Software Engineer